modules/backend/formwidgets/RecordFinder.php

```diff
@@ -299,9 +299,69 @@ public function onFindRecord()
             $this->searchWidget->setActiveTerm(null);
         }
 
+        $this->setParentFormPostKeysToPreserve();
         return $this->makePartial('recordfinder_form');
     }
 
+    /**
+     * If elements from the parent Repeated don't get initialized, this RecordFinder is not initialize,
+     * which means that the searchWidget is not initialized and will not work from a repeated. If values
+     * and keys can be preserved from the parent RecordFinder form when calling the searchWidget, the
+     * RecordFinder will get initialized and will work
+     * @return void
+     */
+    private function setParentFormPostKeysToPreserve(): void
```

Comment:
Private methods are not used in OctoberCMS as protected is better for extensibility most of the time. We also still support PHP 7.0 so `: void` return types can't be used either.